### ISSUE TITLE
Fatal error: Class 'RCP_Payment_Gateway' not found

### ISSUE THREAD

**Issue by benjibee** [restrictcontentpro/library#51]
After copy/pasting the basic example of a custom gateway, and simply changing the name from `Custom` to `Epoch` where appropriate, the following error is thrown when attempting to active the plugin:

```
Fatal error: Class 'RCP_Payment_Gateway' not found in
[…]/wp-content/plugins/rcp_epoch/class-rcp-payment-gateway-epoch.php on line 15
```
Which is coming from `class AG_RCP_Payment_Gateway_Epoch extends RCP_Payment_Gateway`

Before I get into the innards of RCP or spend a while troubleshooting this, is there something obvious I'm missing, or is this snippet no longer relevant?

Running WordPress 5.2.2 and Restrict Content Pro 3.0.9

**Comment by nosegraze** [comment]
@benjibee Does the problem go away if you load your file on `plugins_loaded` ? See this PR for a possible fix: #52

**Comment by benjibee** [comment]
This does indeed fix the issue. I know it's fairly obvious / simple, but I thought it worth bringing it up so this file works as a functional template for others. Thanks for taking the time to submit that PR!
